lib/travis/api/v3/models/repository.rb

```diff
@@ -130,6 +130,10 @@ def admin_settings
       Models::AdminSettings.new(settings).tap { |as| as.sync(self, :settings) }
     end
 
+    def config_validation
```

Comment:
whooops. GitHub didn't tell me about your comments before merging this ...

i think in Ruby it would be `config_validation?`, but ... i think it's fine for now. i don't think we'll have this much longer after enforcing this setting for everyone.